moto/emrserverless/models.py

```diff
@@ -10,14 +10,20 @@
 from moto.core.utils import iso_8601_datetime_without_milliseconds
 from moto.emrcontainers.utils import get_partition, paginated_list
 
-from .exceptions import ResourceNotFoundException, ValidationException
+from .exceptions import (
+    ResourceNotFoundException,
+    ValidationException,
+    AccessDeniedException,
+)
 from .utils import (
     default_auto_start_configuration,
     default_auto_stop_configuration,
     random_appplication_id,
+    random_job_id,
 )
 
-APPLICATION_ARN_TEMPLATE = "arn:{partition}:emr-containers:{region}:{account_id}:/applications/{application_id}"
```

Comment:
There is "emr-containers" instead of "emr-serverless" in a few places, probably because the same contributor did both services originally, and copy/pasted.